README.md

```diff
@@ -108,7 +108,8 @@ The following is a curated list of applications powered by Aleo.
 
 ### Art
 
-- [aleo-nft](https://github.com/demox-labs/aleo-nft) - An NFT collection on the Aleo network
```

Comment:
This is a very early version of nfts on aleo. Art factory has a v3 and v4 program that are much more fully-featured.